src/concrete/ml/onnx/convert.py

```diff
@@ -158,15 +163,21 @@ def get_equivalent_numpy_forward_from_torch(
 
 def get_equivalent_numpy_forward_from_onnx(
     onnx_model: onnx.ModelProto,
-    check_model: bool = True,
+    q_x: numpy.ndarray = None,
+    check_model: Optional[bool] = True,
```

Comment:
check_model is not optional no ?

Reply:
I would say that we always should check the model.

Reply:
no I mean it was originally `check_model: bool = True` but you changed it to `check_model: Optional[bool] = True`, I think that's a mistake no ?